README.md

```diff
@@ -1,18 +1,176 @@
+  for vmlinux and bzImage images as well as some other helpers for kernel
+  commandline.
+- [virtio-bindings](https://github.com/rust-vmm/virtio-bindings): Rust FFI
+  bindings to virtio kernel headers generated using
+  [bindgen](https://crates.io/crates/bindgen).
+- [vm-virtio](https://github.com/rust-vmm/vm-virtio/): virtio device trait and
+  implementation for virtio primitives such as virtqueues and descriptor chain.
+- [vmm-sys-utils](https://github.com/rust-vmm/vmm-sys-util/): a collection of
+  modules providing helpers and utilities for building VMMs and Hypervisors.
+- [vmm-vcpu](https://github.com/rust-vmm/vmm-vcpu/): a hypervisor-agnostic
+  abstraction for Virtual CPUs (vCPUs).
+
+### rust-vmm
+
+- [vm-memory](https://github.com/rust-vmm/vm-memory): abstractions over a
+  Virtual Machine memory.
+
+#### crates.io
+
+- [kvm-bindings](https://crates.io/crates/kvm-bindings): Rust FFI bindings
+  to KVM generated using [bindgen](https://crates.io/crates/bindgen).
+- [kvm-ioctls](https://crates.io/crates/kvm-ioctls): Safe wrappers over the
+  KVM API.
+
+## Development
+
+### Adding a New Virtualization Component
+
+If you have an idea about a component that could be a great fit for rust-vmm,
+please open a
+[review request issue](https://github.com/rust-vmm/community/issues/new?assignees=&labels=&template=new-crate-request.md&title=Crate+Addition+Request).
+The issue must provide the explanations and justifications behind the need for
+a new component, highlighting the purpose of this new repository, how will it
+be used by other VMMs or Hypervisors, and other information that you find
+relevant.
+
+Discussions about the proposal will happen on the same issue, and once an
+agreement is reached regarding the name (this part is hard!) and the design,
+an empty repository will be created in the rust-vmm GitHub organization.
+
+To add functionality you will need to send Pull Requests (PRs) against the
+newly created repository. While we do not expect the code to be perfect from
+the first try, we do want to keep a high standard in terms of quality for all
+existing repositories under rust-vmm.
+
+### Publishing on crates.io - Requirements List
+
+We consider crates published on crates.io to be production ready. In order to
+maintain a high security and quality standard, the following list of
+requirements must be checked before having the first version published:
+
+- **High level documentation**. This documentation should be added
+   to `src/lib.rs` and it will end up being the home page of the crate on
+   [docs.rs](https://docs.rs/). Here you should include information about what
+   is the crate suppose to do, how can it be used in other projects as well as
+   usage examples. Check out the
+   [kvm-ioctls docs](https://docs.rs/kvm-ioctls/0.1.0/kvm_ioctls/) for an
+   example.
+- **Documentation for the public interface**. Everything belonging to the
+  public interface of the crate must be properly documented. Where applicable,
```

Comment:
Maybe link some documentation about what constitutes the public interface of a rust crate?

Reply:
Everything that is exported hence can be used by other crates that take a dependency on it. To me that's obvious and it's also not necessarily rust specific. If it's not obvious for everyone I suppose I can also define that here.

Reply:
Ack. Feel free to resolve (I don't see a button for it).